www/docs/schemas/adapters.md

```diff
@@ -7,6 +7,20 @@ An **Adapter** in NextAuth.js connects your application to whatever database or
 
 You do not need to specify an Adapter explicitly unless you want to use advanced options such as custom models or schemas, if you want to use the Prisma Adapter instead of the default TypeORM Adapter, or if you are creating a custom Adapter to connect to a database that is not one of the supported databases.
 
+The adapters have been migrated out of the core of `nextauthjs/next-auth` into their own repository and can now be found under [`nextauthjs/adapters`](https://github.com/nextauthjs/adapters).
+
+There you can find the following adapters in various stages of maturity:
+
+- TypeORM
+- Prisma
+- Prisma-Legacy
+- FaunaDB
+- DynamoDB
+
+## TypeORM Adapter
+
+NextAuth.js comes with a default Adapter that uses [TypeORM](https://typeorm.io/) so that it can be used with many different databases without any further configuration, you simply add the node module for the database driver you want to use to your project and pass a database connection string to NextAuth.js.
```

Comment:
```suggestion
NextAuth.js comes with a default Adapter that uses [TypeORM](https://typeorm.io/) so that it can be used with many different databases without any further configuration, you simply add the node module for the database driver you want to use in your project and pass a database connection string to NextAuth.js.
```
Super minor but I think this is the correct spelling in English 🤔 ✍🏼

Reply:
Yup you're correct :) I'm a native english speaker and I even missed that :facepalm: haha